feed-notifier.php

```diff
@@ -160,6 +160,25 @@ function deliver_mail_feed ($feed_url) {
 
 }
 
+function delMail($res){
+  if ($res) 
+  { 
+    echo "ok";
+  } else {
+    echo "false";
+  }
+
+  $files = file_get_contents($res);
+
+  return $f;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Review the data-flow - use of uninitialized value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS836)